scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_exome.json

```diff
@@ -1,20 +1,20 @@
 {
-  "case_gatk_acnv_workflow.CalculateTargetCoverage.grouping": "SAMPLE",
+  "case_gatk_acnv_workflow.grouping": "SAMPLE",
   "case_gatk_acnv_workflow.common_snp_list": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/common_snps_sample-chr20.interval_list",
   "case_gatk_acnv_workflow.ref_fasta_fai": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.fasta.fai",
   "case_gatk_acnv_workflow.target_file": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/ice_targets_sample-chr20.tsv",
   "case_gatk_acnv_workflow.gatk_jar": "/home/travis/build/broadinstitute/gatk-protected/build/libs/gatk-protected.jar",
-  "case_gatk_acnv_workflow.PadTargets.mem": "1",
-  "case_gatk_acnv_workflow.CalculateTargetCoverage.disable_all_read_filters": "false",
-  "case_gatk_acnv_workflow.CalculateTargetCoverage.mem": "4",
+  "case_gatk_acnv_workflow.disable_all_read_filters": "false",
+  "case_gatk_acnv_workflow.calc_target_cov_memory": "4",
```

Comment:
I think it's fine to be a little more verbose here and use `calculate_target_coverage_memory`.